src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.re

```diff
@@ -0,0 +1,12 @@
+type t = Dom.domStringMap;
+
+type key = string;
+
+[@bs.get_index] [@bs.return nullable]
+external get: (t, key) => option(string) = "";
+let getPipe: (key, t) => option(string) = (key, map) => get(map, key); 
+[@bs.set_index] external set: (t, key, string) => unit = "";
+let setPipe: (key, string, t) => unit = 
+  (key, value, map) => set(map, key, value);
+let unsafeDeleteKey: (. t, key) => unit =  
+  [%raw (map, key) => "delete map[key];"];
```

Comment:
Argument order needs to be swapped here. It would also be nice if this didn't use uncurrying. I think BuckleScript supports that for this construct.